### PR TITLE
[OPENJDK-1966] October CPU updates.

### PR DESCRIPTION
**Pull request by sefroberg**
Update to the testlib file to not run the base image test on RHEL 7 systems due to lack of support by Podman 1.6.x for the version 2 schema.

Updated version numbers of Java for the October CPU.

Removed code that should not be in there.

### CODE REVIEW

**Review by andrlos — APPROVED** [review]

**Comment by andrlos** [comment]
LGTM, as this is not a big change, all the versions seem to be correct

File: containersQa/testlib.bash [hunk 1]
```diff
@@ -537,20 +545,6 @@ function s2iLocal() {
...
       cat $DF.orig | sed "s;/s2i/run;/s2i/run $ADDS;g;" > $DF.nw1
     fi
-    # simplified, expectig rhel only:
-#    if [ `getOsMajor` -eq 8 ] ; then
-##      cat $DF.nw1 | sed "s|USER root|USER root\nRUN yum config-manager --add-repo 'http://download.eng.brq.redhat.com/composes/finished/latest-RHEL-8/compose/AppStream/\
-##$basearch/os/'\nRUN yum install -y nodejs|" > $DF
-#      #echo
-#      echo "rhel8 repos not added"
-#      #unluckily, config-manager is notworking, removing nodejs from the project itself rather
-#    elif [ `getOsMajor` -eq 7 ] ; then
```
Comment (judovana) [inline containersQa/testlib.bash:547]: Although th https://github.com/rh-openjdk/jdkContainerOsTests/pull/2/files#diff-d4159e621004e9e44da852177057b68c20db675d533dffc305a479f199c911bfR79 is a bit explaining, maybe it hsould be separate commit.

File: containersQa/testlib.bash [hunk 2]
```diff
@@ -702,17 +696,17 @@ function checkHardcodedJdks() {
   if [ "$OTOOL_jresdk" == "jre"  ] ; then
     echo "otool jresdk settings is: $OTOOL_jresdk"
     echo "Check version based off java -version call."
-    JRE_8_VERSION='1.8.0_382-b05'
-    JRE_11_VERSION='11.0.20+8-LTS'
-    JRE_17_VERSION='17.0.8+7-LTS'
+    JRE_8_VERSION='1.8.0_392-b08'
+    JRE_11_VERSION='11.0.21+9-LTS'
+    JRE_17_VERSION='17.0.9+9-LTS'
```
Comment (judovana) [inline containersQa/testlib.bash:701]: should be ok
